panel/chat/langchain.py

```diff
@@ -117,6 +119,7 @@ def on_tool_start(
         self, serialized: Dict[str, Any], input_str: str, *args, **kwargs
     ):
         self._update_active(DEFAULT_AVATARS["tool"], serialized["name"])
+        self._stream(input_str)
```

Comment:
I added `self._stream(input_str)` because its really useful for debugging/ learning to learn what input the tool takes.

For example for the Pandas Agent.

![image](https://github.com/holoviz/panel/assets/42288570/cdcf6ce8-e480-4351-9263-2dc017236205)

Reply:
To clarify to the user what it is, maybe it should be prefixed and wrapped in ``?

Reply:
Maybe. But I don't know if it always makes sense. All I know about input_str is that it is a string?

Would it make sense to do for all tools?

Reply:
Okay maybe let's hold off on ``, but perhaps: `f"Tool input: {input_str}"`?

Reply:
To be consistent then you should do something similar with output `f"Tool output: {...}"`?

Personally I don't think its needed currently. But I believe that when we have learned more about agents and callback usecases we will do a second iteration of the PanelCallbackHandler. Also comparing it systematically to what Streamlit, and Chainlit does. Probably also implementing a second callback handler - for example for HayStack - to understand how a generative AI callback handler should work in general.

Reply:
Should I add it @ahuang11 ?

Reply:
Ok. I comitted your suggestion below.